worlds/ff6wc/__init__.py

```diff
@@ -339,8 +343,35 @@ def generate_early(self):
                 except ValueError as e:
                     raise ValueError(stesp_err_msg) from e
                 if stesp_min > 0 or stesp_max > 0:
-                    # TODO:
-                    raise NotImplementedError("starting espers is not supported")
+                    # pick a random number of starting espers between min & max specified
+                    num_start_espers = random.randint(stesp_min,stesp_max)
```

Comment:
```suggestion
                    num_start_espers = self.random.randint(stesp_min, stesp_max)
```
Global `random` is something we try to avoid because it's global state (https://softwareengineering.stackexchange.com/questions/148108/why-is-global-state-so-evil) and hurts reproducibility when mixing with other worlds.

(I know there is a lot more usage of global `random` here. I have plans to work on getting rid of it.)